parser-util/src/lib.rs

```diff
@@ -50,6 +50,12 @@ pub struct Error {
     message: String,
 }
 
+impl std::fmt::Display for Error {
```

Comment:
Is it a good idea to implement Display like that? We should always try to display it with a proper source reference.

Reply:
The reason it needs `Display` right now its because embedding it into `ParseError::User` requires it.
AFAIU, we won't really show this `Display`, and instead rely on `Error::output_to_stderr`.
I'll make the message be similar to what larlpop does for the other variants of `ParseError`:
`{} at {location}`